tunedev/blog-api
# setup project file structure and define models

#### What does this PR do?
Defines model and set up the server with the apps file structure

#### Description of Task to be completed?
> - add the model definition for User, Post, and Comment
> - add project file structure
> - add server logic and some middlewares

#### How should this be manually tested?
> - Clone this branch
> - Then run `npm install` and `npm test'
> - Then run `npm run start:dev' 
> - server you should get a message in the console that the server has started

#### Any background context you want to provide?
NA

#### Screenshots (if appropriate)
NA
#### Questions:
NA
